ycmd-eldoc.el

```diff
@@ -56,7 +56,7 @@ is only semantic after a semantic trigger."
 
 (defvar-local ycmd-eldoc--cache (make-vector 2 nil))
 
-(defvar-local ycmd-eldoc--get-type-supported-p t)
+(defvar-local ycmd-eldoc--chached-get-type-command 'none)
```

Comment:
I think this should be "cached" instead of "chached". That spelling appears in a few places.